### ISSUE TITLE
#518976679------------- Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1867]
<strong>PageName:</strong> FeedViewDetailsPage  

<strong>Source:</strong>`<div class="esri-header-inlineSearch" aria-expanded="false">`

<strong>Element Location: </strong>.esri-header-inlineSearch

<strong>Impact: </strong>critical

<strong>Failure Summary:</strong> Fix any of the following:
  ARIA attribute is not allowed: aria-expanded="false"

<strong>Tags:</strong> cat.aria,wcag2a,wcag412

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/aria-allowed-attr?application=axeAPI
<strong>Screenshot:<strong>`Couldn't find a screenshot`
